kedro/io/data_catalog.py

```diff
@@ -176,9 +176,9 @@ def __init__(  # noqa: too-many-arguments
         Example:
         ::
 
-            >>> from kedro_datasets.pandas import CSVDataSet
+            >>> from kedro_datasets.pandas import CSVDataset
```

Comment:
As commented in the other PR, these are dependent on the `kedro_datasets` version; do we feel 0.19.0 is a good time to just assume they have a new-enough `kedro_datasets`, or should this be communicated more clearly somewhere?